### PR TITLE
Dockerize

### PR DESCRIPTION
**Pull request by LLParse**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  apt-get update && \
+  apt-get install -y mesos && \
+  rm -rf /var/lib/apt/lists/*
+
+# Build the scheduler
+RUN \
+  git clone https://github.com/mesos/kafka /opt/mesos-kafka && \
+  cd /opt/mesos-kafka && \
+  ./gradlew jar && \
+  curl -O https://archive.apache.org/dist/kafka/0.8.2.1/kafka_2.10-0.8.2.1.tgz
```
Comment (joestein) [inline Dockerfile]: can this be passed through the CLI and the value you have here be the default?

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+RUN \
+  apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv E56151BF && \
+  echo deb http://repos.mesosphere.io/ubuntu trusty main > /etc/apt/sources.list.d/mesosphere.list && \
+  apt-get update && \
+  apt-get install -y mesos && \
+  rm -rf /var/lib/apt/lists/*
+
+# Build the scheduler
+RUN \
+  git clone https://github.com/mesos/kafka /opt/mesos-kafka && \
```
Comment (joestein) [inline Dockerfile]: If they have this Dockerfile then they have the repo, shouldn't/wouldn't the location of the /opt/mesos-kafka be a volume mounted in or something of the sort?

File: docker-cli.sh [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+        echo "$jar not found"
+        exit 1
+    elif [ $jars -gt 1 ]
+    then
+        echo "More than one $jar found"
+        exit 1
+    fi
+}
+
+check_environment() {
```
Comment (joestein) [inline docker-cli.sh:34]: This looks great, can you update the README with a section about all of this and how it works please? Thanks!

File: Dockerfile (hunk 1 above)
Comment (LLParse) [inline Dockerfile]: Yes, I will add environment variables KAFKA_VERSION and SCALA_VERSION and add logic to download the correct version when the container launches. Down the road we could tag versions and have the correct binary baked into an fslayer, overkill for now.

File: Dockerfile (hunk 2 above)
Comment (LLParse) [inline Dockerfile]: Good point, I should really just be using a command like 'ADD . /opt/mesos-kafka'. That will make it easier for developers to bake their own containers with local changes.

File: docker-cli.sh (hunk 3 above)
Comment (LLParse) [inline docker-cli.sh:34]: Certainly.

**Comment by joestein** [comment]
taken care of in #35

**Pull request closed without merging**